gateware/dragonBoot/flash.py

```diff
@@ -180,6 +185,7 @@ def elaborate(self, platform) -> Module:
 					m.d.sync += [
 						op.eq(SPIFlashOp.erase),
 						byteCount.eq(self.byteCount),
+						enableStep.eq(0),
```

Comment:
This assignment should be duplicated by the one on line 171 - it should be able to be dropped.